sync/linux/smoke-test.yaml

```diff
@@ -4,62 +4,62 @@ kind: Deployment
 metadata:
     name: windows-server-iis
 spec:
-    selector:
-        matchLabels:
-            app: windows-server-iis
-    replicas: 1
-    template:
-        metadata:
-            labels:
-                app: windows-server-iis
-        spec:
-          tolerations:
-          - effect: NoSchedule
-            key: os
-            operator: Equal
-            value: windows
-          - effect: NoSchedule
-            key: os
-            operator: Equal
-            value: Windows    
-          containers:
-          - name: windows-server-iis
-            image: mcr.microsoft.com/windows/servercore:1809
-            imagePullPolicy: IfNotPresent
-            command:
-            - powershell.exe
-            - -command
-            - "Add-WindowsFeature Web-Server; Invoke-WebRequest -UseBasicParsing -Uri 'https://dotnetbinaries.blob.core.windows.net/servicemonitor/2.0.1.6/ServiceMonitor.exe' -OutFile 'C:\\ServiceMonitor.exe'; echo '<html><body><br/><br/><marquee><H1>Hello EKS!!!<H1><marquee></body><html>' > C:\\inetpub\\wwwroot\\default.html; C:\\ServiceMonitor.exe 'w3svc'; "
-          nodeSelector:
-            kubernetes.io/os: windows
+  selector:
+    matchLabels:
+        app: windows-server-iis
+  replicas: 1
+  template:
+    metadata:
+      labels:
+          app: windows-server-iis
+    spec:
+      tolerations:
+      - effect: NoSchedule
+        key: os
+        operator: Equal
+        value: windows
+      - effect: NoSchedule
+        key: os
+        operator: Equal
+        value: Windows    
+      containers:
+      - name: windows-server-iis
+        image: mcr.microsoft.com/windows/nanoserver:1809
+        imagePullPolicy: IfNotPresent
+        command:
+        - powershell.exe
+        - -command
+        - "Add-WindowsFeature Web-Server; Invoke-WebRequest -UseBasicParsing -Uri 'https://dotnetbinaries.blob.core.windows.net/servicemonitor/2.0.1.6/ServiceMonitor.exe' -OutFile 'C:\\ServiceMonitor.exe'; echo '<html><body><br/><br/><marquee><H1>Hello EKS!!!<H1><marquee></body><html>' > C:\\inetpub\\wwwroot\\default.html; C:\\ServiceMonitor.exe 'w3svc'; "
```

Comment:
wip - still finding the correct command here

Reply:
you could use something like stefanscherer/whoami:windows-amd64-2.0.1 which is based on nano so it would be fast install (I assume that is why you don't use the iis image directly)

Reply:
https://github.com/jsturtevant/windows-k8s-playground/blob/master/deployments/whoami-windows.yaml